addons/phone_validation/models/mail_thread_phone.py

```diff
@@ -47,6 +50,26 @@ class PhoneMixin(models.AbstractModel):
             when there is both a mobile and phone field in a model.")
     phone_mobile_search = fields.Char("Phone/Mobile", store=False, search='_search_phone_mobile_search')
 
+    def init(self):
+        super().init()
+        phone_fields = [
+            fname for fname in self._phone_get_number_fields()
+            if fname in self._fields and self._fields[fname].store
+        ]
+        for fname in phone_fields:
+            regex_expression = rf"regexp_replace(({fname}::text), '{PHONE_REGEX_PATTERN}'::text, ''::text, 'g'::text)"
+            create_index(self.env.cr,
```

Comment:
Would maybe add a small comment explaining the usage of each

Reply:
Done :)